### ISSUE TITLE
Travis Nix job exceeded the maximum log length

### ISSUE THREAD

**Issue by jamesdbrock** [gibiansky/IHaskell#1044]
The Nix builds are failing on Travis because

```
The job exceeded the maximum log length, and has been terminated.
```

**Comment by jamesdbrock** [comment]
I see you've been working on the Nix build @vaibhavsagar 

https://github.com/gibiansky/IHaskell/pull/852

**Comment by vaibhavsagar** [comment]
Closed by #1103.
